src/services/enrollment.service.ts

```diff
@@ -5,18 +5,14 @@ import { type EnrollStatus, type PostEnroll } from '@/types/enrollments.types';
 
 export const getStudentEnrollments = async (
 	index: string,
+	enrollStatuses: EnrollStatus[],
 	userId?: number,
 	semesterId?: number,
-	enrollStatuses?: EnrollStatus[],
 ) => {
 	let data;
-	let statuses = '';
-	if (enrollStatuses) {
-		statuses += `&statuses=${enrollStatuses[0]}`;
-		for (let i = 1; i < enrollStatuses.length; i++) {
-			statuses += `%2C${enrollStatuses[i]}`;
-		}
-	}
+	const statuses = enrollStatuses
+		? `&statuses=${enrollStatuses.join('%2C')}`
+		: '';
```

Comment:
To samo. CTRL+C/V
nadal nie wiem czemu tutaj robisz `%2C `zamiast po prostu `,`
Wydaje mi się, że next powinien ogarnąć i sam zamienić `,` na`%2C` a nawet jeśli nie to niżej przy budowaniu linku możesz dać `encodeURIComponent()`